app/views/homepage/index.html.erb.orig

```diff
@@ -0,0 +1,28 @@
+
```

Comment:
Delete this whole file (index.html.erb.orig). Mergetool made it, but we don't need it anymore.